packages/store/src/services/backend-system.ts

```diff
@@ -0,0 +1,66 @@
+import { Logger } from '@sap-ux/logger';
+import { Service } from '.';
+import { DataProvider } from '../data-provider';
+import { SystemDataProvider } from '../data-provider/backend-system';
+import { BackendSystem, BackendSystemKey } from '../entities/backend-system';
+import { text } from '../i18n';
+import { ServiceOptions } from '../types';
+
+export class SystemService implements Service<BackendSystem, BackendSystemKey> {
```

Comment:
jsdoc

Reply:
https://github.com/SAP/open-ux-tools/pull/237#discussion_r791175087

Reply:
Added a comment in the interface in https://github.com/SAP/open-ux-tools/pull/237/commits/e37e1de9ea46e33d3a9e87c4febfd4187ff5186a